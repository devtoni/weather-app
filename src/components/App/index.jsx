import React from 'react'
import 'normalize-css'

import Routes from '../../routes'
import styles from './app.scss'

const App = () => {
  return (
    <Routes />
  )
}

export default App
