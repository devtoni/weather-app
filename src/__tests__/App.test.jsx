import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import App from '../components/App/'

test('App component should render as expected', () => {
  const component = shallow(<App />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
