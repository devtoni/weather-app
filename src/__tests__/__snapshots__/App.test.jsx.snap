// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render as expected 1`] = `<Routes />`;
